src/gsy_e/constants.py

```diff
@@ -37,7 +38,7 @@
 DISPATCH_EVENT_TICK_FREQUENCY_PERCENT = 10
 DISPATCH_MYCO_EVENT_TICK_FREQUENCY_PERCENT = 2
 
-CONFIGURATION_ID = ""
+CONFIGURATION_ID = os.environ.get("CONFIGURATION_ID", "")
```

Comment:
I was worried at first that the `rq_job_handler` would be influenced by this in the rare case when this variable is set in the environment and we start the launcher. But AFAIU, this defined at startup and the rq_job_handler will overwrite this value. 
Nothing to do here, just thinking out loud and hope that I am right. Thanks!